src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1624,7 +1641,7 @@ local function find_room(name, area)
         local areas = getAreaTable() or {}
         for k,v in pairs(areas) do
             if string.lower(k) == string.lower(area) then
```

Comment:
*\<aside>This looks suspect to me - the main C++ core handles Area **names** in a case sensitive manner - so for Lua functions to discard the casing could lead to some awkward corner case behaviour...!\</aside>*

Reply:
this is not in the PR

Reply:
Agreed, but it is something that those who look after this need to check - unfortunately it isn't really a strong area (something I know a lot about) for me.